src/main/java/String10.java

```diff
@@ -0,0 +1,33 @@
+public class String10 {
+    public String sum(String a, String b) {
+        int lenA = a.length();
+        int lenB = b.length();
```

Comment:
Ở đây khai báo lenA, lenB và vì nó có thể được sử dụng trong toàn bộ hàm nên nếu vô tình dùng sau lệnh if thì sẽ bị sai kết quả. (Vì bị lỗi thời). Vậy nên bỏ 2 biến này